astropaint/paint_bucket.py

```diff
@@ -2394,7 +2395,9 @@ def snap2grid(array, grid):
 
             # count the number of available cpus
             # import psutil
```

Comment:
this `import psutil` can be also removed. Sorry, I just left it there!

Reply:
you can also remove both commented arguments `#with_ray=False` and `#parallel=True` from the function.